<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setting</class>
 <widget class="QWidget" name="setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="Watt_symbol_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>300</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Up_button">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>180</y>
     <width>121</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>▲</string>
   </property>
  </widget>
  <widget class="QLabel" name="Temp_Watt_symbol_label">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>20</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLCDNumber" name="TimeSLCD">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>430</y>
     <width>111</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="smallDecimalPoint">
    <bool>false</bool>
   </property>
   <property name="digitCount">
    <number>2</number>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>80</y>
     <width>1041</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="colorr_label">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Temp_symbol">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>130</y>
     <width>101</width>
     <height>112</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ºc</string>
   </property>
  </widget>
  <widget class="QLabel" name="Modelabel">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Continuance</string>
   </property>
  </widget>
  <widget class="QLabel" name="colorr_label_2">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Time_symbol">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>450</y>
     <width>151</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Min | Sec</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="BackButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
   <property name="autoRepeatDelay">
    <number>300</number>
   </property>
  </widget>
  <widget class="QLabel" name="Temp_symbol_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Time_symbol_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>480</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLCDNumber" name="TempLCD">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>120</y>
     <width>161</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="smallDecimalPoint">
    <bool>false</bool>
   </property>
   <property name="digitCount">
    <number>3</number>
   </property>
   <property name="mode">
    <enum>QLCDNumber::Dec</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="TimeMLCD">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>430</y>
     <width>121</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="smallDecimalPoint">
    <bool>false</bool>
   </property>
   <property name="digitCount">
    <number>2</number>
   </property>
  </widget>
  <widget class="QPushButton" name="Down_button">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>370</y>
     <width>121</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>▼</string>
   </property>
  </widget>
  <widget class="Line" name="Temp_line">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>220</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="TimeLCD_label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>430</y>
     <width>111</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="smallDecimalPoint">
    <bool>false</bool>
   </property>
   <property name="digitCount">
    <number>2</number>
   </property>
  </widget>
  <widget class="QPushButton" name="ModeButton">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>10</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>        Mode</string>
   </property>
  </widget>
  <widget class="QLabel" name="Watt_symbol">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>290</y>
     <width>111</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>W</string>
   </property>
  </widget>
  <widget class="Line" name="Watt_line">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>350</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="TimeM_line">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>530</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="WattLCD">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>250</y>
     <width>161</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="smallDecimalPoint">
    <bool>false</bool>
   </property>
   <property name="digitCount">
    <number>3</number>
   </property>
  </widget>
  <widget class="Line" name="TimeS_line">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>530</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <zorder>ModeButton</zorder>
  <zorder>Watt_symbol_label</zorder>
  <zorder>Up_button</zorder>
  <zorder>Temp_Watt_symbol_label</zorder>
  <zorder>TimeSLCD</zorder>
  <zorder>line</zorder>
  <zorder>colorr_label</zorder>
  <zorder>Temp_symbol</zorder>
  <zorder>Modelabel</zorder>
  <zorder>colorr_label_2</zorder>
  <zorder>Time_symbol</zorder>
  <zorder>BackButton</zorder>
  <zorder>Temp_symbol_label</zorder>
  <zorder>Time_symbol_label</zorder>
  <zorder>TempLCD</zorder>
  <zorder>TimeMLCD</zorder>
  <zorder>Down_button</zorder>
  <zorder>Temp_line</zorder>
  <zorder>TimeLCD_label</zorder>
  <zorder>Watt_symbol</zorder>
  <zorder>Watt_line</zorder>
  <zorder>TimeM_line</zorder>
  <zorder>WattLCD</zorder>
  <zorder>TimeS_line</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
